capa/render/default.py

```diff
@@ -18,6 +18,7 @@
 from capa.render.utils import StringIO
 
 tabulate.PRESERVE_WHITESPACE = True
+MIN_LIBFUNCS_COUNT = 5
```

Comment:
I am indecisive on what to choose here. According to this [documentation](https://hex-rays.com/products/ida/tech/flirt/in_depth/):
>our analysis has shown that real life programs contain, on average, 50% of library functions.

Should we base this off of this conclusion?

Reply:
50% is a good estimate, maybe we want to be a little more conservative (40%?) here first